build-tools/packages/build-tools/src/fluidBuild/buildResult.ts

```diff
@@ -0,0 +1,24 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+export enum BuildResult {
+	Success,
+	UpToDate,
+	Failed,
+}
+
+export function summarizeBuildResult(results: BuildResult[]) {
```

Comment:
I had to split this out into its own file, since there is a cyclic dependency (leaf task depended on buildGraph which pulled in all the tasks). I guess the production code managed to load things in an order that worked, but adding the new test file caused the tests to crash due the cycle failing to get resolved.

Reply:
Docs? 😁

Reply:
It had no docs before, and I didn't modify it, but I suppose I can try and reverse engineer what its for.